roles/ansible-service-broker-setup/tasks/main.yml

```diff
@@ -0,0 +1,57 @@
+---
+
+- name: Verify Ansible Service Broker project name exists
+  block:
+    - name: Get Ansible Service Broker project name
+      shell: oc get projects | grep -e ansible-service-broker -e automation-service-broker | awk '{print $1}'
+      register: broker_project_name
+      failed_when: broker_project_name.stdout == ""
+    - set_fact:
+        broker_project_name: '{{ broker_project_name.stdout }}'
+  rescue:
+    - name: Fail when Ansible Service Broker project was not found
+      pause:
+        prompt: |
+          Ansible Service Broker project not found.
+          Please verify that currently logged in user has privileges to list and edit Ansible Service Broker Project.
+
+          Hit Ctrl + c and then 'a' to abort the installation
```

Comment:
I think if we move this role to the first place in the playbook [here](https://github.com/aerogear/mobile-services-installer/pull/1/files#diff-33307e400add9796d9bdbaea915a1cdbR5), it could actually do the thing as well, because this task will become the first one that is run. wdyt?

Reply:
@psturc yes, you are right, that's even simpler. +1 from me.